system/jhtree/jhtree.cpp

```diff
@@ -673,6 +670,27 @@ class CNodeMRUCache final : public CMRUCacheOf<CKeyIdAndPos, CNodeCacheEntry, CN
     std::atomic<size32_t> sizeInMem{0};
     size32_t memLimit = 0;
 public:
+    std::shared_ptr<hpccMetrics::CounterMetric> numHits = nullptr;
+    std::shared_ptr<hpccMetrics::CounterMetric> numAdds = nullptr;
+    std::shared_ptr<hpccMetrics::CounterMetric> numDups = nullptr;
+    std::shared_ptr<hpccMetrics::CounterMetric> numEvicts = nullptr;
+    bool enabled = false;
+    CNodeMRUCache(CacheType cacheType)
+    {
+        StringBuffer name, desc;
+        name.appendf("jhtree.cache.%s.hits", cacheTypeText[cacheType]);
+        desc.appendf("The number of cache hits on the jhtree %s cache", cacheTypeText[cacheType]);
+        numHits = hpccMetrics::registerCounterMetric(name, desc, SMeasureCount);
+        name.appendf("jhtree.cache.%s.adds", cacheTypeText[cacheType]);
+        desc.appendf("The number of cache adds to the jhtree %s cache", cacheTypeText[cacheType]);
+        numAdds = hpccMetrics::registerCounterMetric(name, desc, SMeasureCount);
+        name.appendf("jhtree.cache.%s.dupes", cacheTypeText[cacheType]);
+        desc.appendf("The number of cache add collisions on the jhtree %s cache", cacheTypeText[cacheType]);
+        numDups = hpccMetrics::registerCounterMetric(name, desc, SMeasureCount);
+        name.appendf("jhtree.cache.%s.evicted", cacheTypeText[cacheType]);
```

Comment:
should probably be .evictions for consistency.  Alternatively .added, .deduped, .matched or similar.

Reply:
done